File: JavaScript/Exercicios - FACCAT/Exercicio-9/exercicio09.js
// Exercícios, Lógica de Programação, Algoritmos e JavaScript
// Escreva um algoritmo para ler o salário mensal atual de um funcionário e o percentual de reajuste. Calcular e escrever o valor do novo salário. 

// Campo do exercício 09
let salario, novoSalario, reajuste, resultado; //variável

alert("Calculando o reajuste do salário atual") //saída de dados

salario=parseFloat(prompt("Digite o valor do seu salário atual: ")); //entrada e saída de dados
reajuste=parseFloat(prompt("Digite a porcentagem do reajuste: ")); //entrada e saída de dados

resultado = reajuste/100*salario //Processamento
novoSalario = resultado+salario //Processamento

alert("O salário com o reajuste é: "+novoSalario) //saída de dados

File: Java/Exercicios - FACCAT/Exercicio09.java
package javaBasico;

import java.util.Scanner;

public class Exercicio09 {
	
	public static void main(String[] args) {
		
		Scanner dados = new Scanner(System.in);
		
		 System.out.println("Calculando o reajuste do salário atual");
		 
		 System.out.println("Digite o valor do seu salário atual: ");
		 float salario = dados.nextFloat();
		 
		 System.out.println("Digite a porcentagem do reajuste: ");
		 float reajuste = dados.nextFloat();
		 
		
		 float resultado = reajuste/100*salario;
		 float novoSalario = resultado+salario;
		 
		 System.out.println("O salário com o reajuste é: "+novoSalario);
		 
		 dados.close();
	}
}

// Exercícios, Lógica de Programação, Algoritmos e JavaScript

/* Escreva um algoritmo para ler o salário mensal atual de um funcionário e o percentual de reajuste. Calcular e escrever o valor do novo salário. */
